tests/components/rainforest_raven/test_config_flow.py

```diff
@@ -282,9 +254,7 @@ async def test_flow_user_invalid_type(
     assert result.get("flow_id")
     assert result.get("step_id") == "meters"
 
-    result = await hass.config_entries.flow.async_configure(
-        result["flow_id"], user_input={CONF_MAC: mac}
-    )
-    assert result
-    assert result.get("type") == data_entry_flow.FlowResultType.FORM
-    assert result.get("errors") == {CONF_MAC: "no_meters_found"}
+    with pytest.raises(voluptuous.error.MultipleInvalid):
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"], user_input={CONF_MAC: ["abcdef1234567890"]}
+        )
```

Comment:
When can this happen?

Reply:
I'd suppose it no longer can. I was just adapting a previous test case to work correctly under the `SelectSelector` change.

I'll go ahead and drop it now.

Reply:
Done in 19bc827.